src/game/client/draw/drawable.cpp

```diff
@@ -174,11 +180,17 @@ void Drawable::Set_Position(Coord3D const *pos)
 
 ClientUpdateModule *Drawable::Find_Client_Update_Module(NameKeyType key)
 {
-#ifdef GAME_DLL
-    return Call_Method<ClientUpdateModule *, const Drawable, NameKeyType>(PICK_ADDRESS(0x00477A80, 0x007C9F40), this, key);
-#else
+    ClientUpdateModule **modules = Get_Client_Update_Modules();
+
+    if (modules != nullptr) {
+        while (*modules != nullptr) {
```

Comment:
The original code looks like this. Definitely a while loop and definitely no increment.

Reply:
Unless we have evidence to suggest that what is being done is a bug (rather than something intentional) we should not diverge from how the original did it IMO.